util/dockerfiles/master/gasnet/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM chapel/chapel:master
```

Comment:
Not a complaint, but whether this FROM line would work as intended depends on how the master-branch non-gasnet Chapel Docker image gets named after you build it; that image name is not defined in its Dockerfile. Instead, you name the image after you build it, with another `docker` command. At least, that's how I learned it.

Reply:
I copied https://github.com/chapel-lang/chapel/blob/master/util/dockerfiles/1.17.0/Dockerfile and it doesn't to a ``make cleanall`` either? A quick scan of 1.16 and 1.15 don't do it either.

The ``chapel/chapel:master`` was a copy & edit from 1.17 as well -- I just renamed ``latest`` to ``master``

FWIW -- I've only run the non-gasnet version to make https://hub.docker.com/r/pnnl/chapel/

Reply:
Ah, I see what I did. The cleanall isn't in the Dockerfiles in the versioned subfolders but is in the root.